charts/prometheus/templates/alertmanager/deploy.yaml

```diff
@@ -60,7 +60,12 @@ spec:
           args:
             - --config.file=/etc/config/{{ .Values.alertmanager.configFileName }}
             - --storage.path={{ .Values.alertmanager.persistentVolume.mountPath }}
+          {{- if .Values.alertmanager.service.enableMeshPeer }}
+            - --cluster.listen-address=0.0.0.0:6783
             - --cluster.advertise-address=[$(POD_IP)]:6783
+          {{- else }}
+            - --cluster.listen-address=
```

Comment:
Should this be empty?

Reply:
It’s to turn off HA on Alertmanager

ref: https://github.com/prometheus/alertmanager#turn-off-high-availability

Reply:
@monotek  any comments ?